apps/admin_api/config/config.exs

```diff
@@ -9,7 +9,8 @@ use Mix.Config
 config :admin_api,
   namespace: AdminAPI,
   ecto_repos: [],
-  sender_email: System.get_env("SENDER_EMAIL") || "admin@localhost"
+  sender_email: System.get_env("SENDER_EMAIL") || "admin@localhost",
+  enable_client_auth: System.get_env("ENABLE_ADMIN_CLIENT_AUTH") || "false"
```

Comment:
Can we convert "true" to boolean here so we don't need to worry about string vs. boolean downstream? E.g.

```ex
  enable_client_auth: System.get_env("ENABLE_ADMIN_CLIENT_AUTH") == "true" || false
```